Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -406,7 +406,7 @@ public void DropDownList ()
       }
       else if (Helper.BrowserConfiguration.IsFirefox())
       {
-        padding = new WebPadding (2, 1, 2, 68);
+        padding = new WebPadding (6, 1, 2, 68);

```

Comment:
The width of an empty dropdown changed.

![image](https://user-images.githubusercontent.com/40357550/93065192-b4ea8c00-f678-11ea-88ef-a3acad55bb6e.png)